### DISCUSSION TITLE
添加自动切换到原版直播间功能

### DISCUSSION THREAD

**Discussion by shoucandanghehe** [the1812/Bilibili-Evolved#2133] [功能建议 / Ideas]
![image](https://user-images.githubusercontent.com/51957264/128852594-e4145035-1378-425c-ab7d-dda73b1fdb8e.png)
像这种活动直播间用起来很难受，少了很多东西
在房间号前加 _blanc_ 可以切换回原版
`https://live.bilibili.com/blanc/roomid`
但是每次手动切很麻烦，能不能加个自动切原版直播间的功能

**Comment by lostpg** [comment]
活动直播间里很多脚本都没法正常运行，而且内存占用比原版高了很多。希望增加一个快捷切换的功能按钮。
